package.json

```diff
@@ -12,45 +12,46 @@
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
+    "svg-images": "svgo -rf .",
```

Comment:
if this command need to be ran always before build
we should add it to the build command like:
```
"build": "yarn svg-images && docusaurus build"
```
from reading `svg-images` i don't understand what this command do, can you explain what it does exactly?